pkg/doc/verifiable/credential_test.go

```diff
@@ -747,27 +757,30 @@ func TestCredentialSubjectId(t *testing.T) {
 					"spouse": "did:example:ebfeb1f712ebc6f1c276e12ec21",
 				},
 			}}
-		_, err := subjectID(vcWithMultipleSubjects.Subject)
+		sID, err := subjectID(vcWithMultipleSubjects.Subject)
```

Comment:
Same thing here. This is me nitpicking though, so consider these "s" short form changes optional - I'm okay with whatever you decide here

Reply:
OK